chcemvediet/urls.py

```diff
@@ -13,9 +13,11 @@
 
 # Initializes the project
 from . import ready
+from .sitemaps import sitemaps
 
 
 urlpatterns = patterns(u'',
+    url(r'^sitemap[.]xml$', u'django.contrib.sitemaps.views.sitemap', kwargs=dict(sitemaps=sitemaps)),
     url(r'^mandrill/', include(u'poleno.mail.transports.mandrill.urls', namespace=u'mandrill')),
     url(r'^styleguide/$', TemplateView.as_view(template_name=u'styleguide/main.html'), name=u'styleguide'),
     url(r'^i18n/', include(u'django.conf.urls.i18n')),
```

Comment:
Este som zabudol, ze aj app `obligees` nam generuje jeden verejny view `obligees:index`. Aj pren treba spravit sitemap. Pozor na to, ze na stranka je pagovana. Do sitemapy by sme mali dat linky na vsetky jej page. Vid view `index` v `chcemvediet/apps/obligees/views.py`

Reply:
:ok: